Specs/Renderer/TextureSpec.js

```diff
@@ -1,6 +1,7 @@
 import { Cartesian2 } from "../../Source/Cesium.js";
 import { Color } from "../../Source/Cesium.js";
-import { loadKTX } from "../../Source/Cesium.js";
+import { loadKTX2 } from "../../Source/Cesium.js";
+import KTX2Transcoder from "../../Source/Core/KTX2Transcoder.js";
```

Comment:
```suggestion
import { KTX2Transcoder } from "../../Source/Core/KTX2Transcoder.js";
```

Reply:
whoops sorry for the incomplete fix (CC https://github.com/CesiumGS/cesium/pull/9513/commits/938a742928af2701b410149a99c4e3b081ec1d48)